### DISCUSSION TITLE
Save Page Style Locally rather than Globally

### DISCUSSION THREAD

**Discussion by Kinasho** [xgi/houdoku#143] [Feature Requests]
Currently, Houdoku saves Page Styles globally, meaning that for example, if I was reading Berserk in Single Page mode, then read Tower of God in Long Strip mode, when I go back to Berserk, it would show in Long Strip Mode.

This can be an annoyance after a while, so a better alternative would be to save Page Styles locally for each manga, whereby manga that are switched from the Default Page Style are saved locally, meaning that if the situation above were to happen, Berserk would save in Single Page mode, and if I were to go back to Tower of God, it would be in Long Strip mode.

(Also sorry for putting so many feature requests. I really like this app, and would love to see it get better over time since I know from experience that feedback is the most important thing for improvement)

**Comment by xgi** [comment]
I can see the merit for this, but I'm not sure what the UX would be like for differentiating between a local and global change. If the default is SinglePage and someone wanted to read all of their series in LongStrip, they may open a chapter, change to LongStrip, but then when they go to read another series it would be back to SinglePage.

Regardless this would probably end up requiring some kind of series-specific override, and it would be a hassle development-wise to basically add settings info to series data.

**Reply by CheekyNinja8** [reply]
With the new UX changes in update v2.16 I feel implementing this feature is more straighforward than before :)

You could repurpose the page style and page fit buttons that now appear in the sidebar as manga specific settings, while kepping the options in the settings tab as the default settings. 
![image](https://github.com/user-attachments/assets/b27c7886-bdb8-42b7-8795-89dd6e93068a)

**Reply by WilliShakes** [reply]
An other possibility could be to use presets. Like that you don't need to implement it for every series. I would even be okay to edit those presets outside of Houdokou if it's really complicated to adapt the UX, since it's not something I would need to change often anymore (but of course that's only my opinion).
